src/app/components/BibPage/DefinitionList.jsx

```diff
@@ -7,28 +7,29 @@ import SubjectHeadings from './SubjectHeadings';
  * Expects data in the form of [{ term: '', definition: '' }, {...}, ...].
  */
 const DefinitionList = ({ data, headings }) => {
-  const getDefinitions = (definitions) =>
-    definitions.map((item) => {
-      if (!item || (!item.term && !item.definition)) {
-        return null;
-      }
-
-      if (item.term === 'Subject' && headings)
-        return <SubjectHeadings key='subjects' headings={headings} />;
-
-      return [
-        <dt key={`term-${item.term}`}>{item.term}</dt>,
-        <dd data={`definition-${item.term}`} key={`definition-${item.term}`}>
-          {item.definition}
-        </dd>,
-      ];
-    });
-
   if (!data || !data.length) {
     return null;
   }
 
-  return <dl>{getDefinitions(data)}</dl>;
+  return (
+    <dl>
+      {data.map((item) => {
+        if (!item || (!item.term && !item.definition)) {
+          return null;
+        }
+
+        if (item.term === 'Subject' && headings)
+          return <SubjectHeadings key='subjects' headings={headings} />;
```

Comment:
So this is interesting. I'm not entirely sure if by passing `key` to a defined component works the same way as passing a `key` to an html element. The `key` would techinically be a prop to the `SubjectHeadings` component, wouldn't it? But, in terms of following the conventions of the `key` property, yes, it would most definitely be a duplicate. However, you'll notice the `item.term` being compared on line `21`. Though it's not clear from this file, there is a single `item.term` which contains the value of `Subject` and this would not create a duplicate. But to avoid the potential of duplicates, I'll make a change to reflect the React convention. How about using an index value?

Reply:
So, I have validated that this particular case where a hard coded 'subjects' key is being sent down to the SubjectHeadings Component is only going to be used once and not actually passed down to the SubjectHeadings Component. Additionally, the item.term, which is checked against the value 'Subject', comes from the field array in BottomBibDetails. It is only set once. So, this means 'subject' will not ever be a duplicate key. Adding context around the key in unnecessary.